cli/src/main.rs

```diff
@@ -230,6 +230,25 @@ fn main() -> Result<(), Box<dyn error::Error>> {
     )
     .get_matches();
 
+    if let Err(e) = do_main(&matches) {
+        use thiserror::Error;
+
+        #[derive(Error)]
+        #[error("")]
+        struct PrettyError(String);
+
+        impl std::fmt::Debug for PrettyError {
+            fn fmt(&self, fmt: &mut std::fmt::Formatter) -> std::fmt::Result {
+                write!(fmt, "{}", self.0)
+            }
+        }
```

Comment:
Any reason this stuff needs to be in `main()`? Not really consistent with how we typically structure things, esp the use statement.

Reply:
Nah.  I was just in an "oh shit!" frenzy when I realized I broke return codes at the last minute.

I'll move it into `clap-utils` and give it a better name

Reply:
lol, I did the same thing when I tried to sneak a display->debug change into another cli pr